runtime/gc_vlhgc/CopyForwardScheme.cpp

```diff
@@ -3540,7 +3540,7 @@ MM_CopyForwardScheme::scanContinuationObjects(MM_EnvironmentVLHGC *env)
 						}
```

Comment:
I think we should add `else` here and use assertion with message to print `pointer`

Reply:
there is an example of assertion with message:
```
	Assert_GC_true_with_message(env, false, "Continuation object  %p should be marked, but it is not\n", pointer);
```

Reply:
but if pointer is "dead object", _markMap->isBitSet(pointer) should be false.

Reply:
Correct. Message can be "dead continuation object"

Reply:
ah, sorry, don't mind

Reply:
A couple of minor things (remnants of finalizable code):
- remove: commented a line above: bool finalizable = false;
- remove or update a comment below: .... it is now finalizable so push it to the local buffer